ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/RawBatch.scala

```diff
@@ -127,6 +136,12 @@ private[events] object RawBatch {
           "exercise_consuming" -> exercise.consuming,
           "exercise_choice" -> exercise.choiceId,
           "exercise_actors" -> exercise.actingParties.toArray[String],
+          // set create event columns to NULL
+          "create_argument" -> nullParamValue[Array[Byte]],
+          "create_signatories" -> nullParamValue[Array[String]],
+          "create_observers" -> nullParamValue[Array[String]],
+          "create_agreement_text" -> nullParamValue[String],
+          "create_key_value" -> nullParamValue[Array[Byte]],
```

Comment:
Doesn't `None` work?

Reply:
`None` does not work, it needs a type in order to set column to null:
```
s.setNull(index, jdbcType)
```